savlatkhon-eshonov-1/task-3.ts

```diff
@@ -0,0 +1,47 @@
+interface Book {
+  title: string;
+  author: string;
+  isbn: number;
+}
+
+class Library {
+  books: Book[]
```

Comment:
you don't need to have interface for Book. For library class books:Book[]. (Book must be class Book not interface)